verification/cbmc/sources/make_common_data_structures.c

```diff
@@ -30,6 +31,14 @@
 #include <proof_helpers/make_common_data_structures.h>
 #include <proof_helpers/proof_allocators.h>
 
+struct default_cmm {
+    struct aws_cryptosdk_cmm base;
+    struct aws_allocator *alloc;
+    struct aws_cryptosdk_keyring *kr;
+    // Invariant: this is either DEFAULT_ALG_UNSET or is a valid algorithm ID
```

Comment:
Avoid cpp comments.
```c
/* Invariant: this is either DEFAULT_ALG_UNSET or is a valid algorithm ID. */
```

Reply:
I copied this from the source code, so I fixed it in both places.